### PR TITLE
React-Redux-Capstone

### PR DESCRIPTION
**Pull request by Kgomotso196**
# PROJECT TO DOs
### IN THIS PROJECT I:

1. Used React documentation.
2. Used React components.
3. Used React props.
4. Used React Router.
5. Connected React and Redux.
6. Handled events in a React app.
7. Wrote integration tests with a React testing library.
8. Used styles in a React app.
9. Used React life cycle methods.
10. Applied React best practices and language style guides in code.
11. Used store, actions and reducers in React.
12. Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
13. Made sure that I used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
14. Made sure that I documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,231 +25,144 @@ For more information on the importance of a professional README for your reposit
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
-
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to express my heartfelt gratitude to Microverse for providing me with this incredible project opportunity.
```
Comment (descholar-ceo) [inline README.md:147]: - [ ] The README should contains an "Acknowledgements" section or similar that gives appropriate credit to Nelson Sakwa the author of the original design, as required by the Creative Commons License.

File: README.md [hunk 2]
```diff
@@ -61,231 +25,144 @@ For more information on the importance of a professional README for your reposit
...
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+- npx hint .
+- npx stylelint "\*_/_.{css,scss}"
```
Comment (descholar-ceo) [inline README.md]: - [ ] These commands don't run tests, to run tests you use `npm test` the ones you showed are used to run linters instead

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @Kgomotso196 🤝 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
👍🏽 The correct usage of git flow ✔️ 
👍🏽 The PR has a title and a description ✔️ 
👍🏽 The linters is isntalled and the passes the check ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] I noticed that you have written less test cases, would you please test as many as possible components ?

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Kgomotso196) [reply README.md]: @descholar-ceo , I will have that fixed. Thanks.

**Review by amejid — APPROVED** [review]
Hi @Kgomotso196, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linter checks passing ✔️ 
- Changes implemented ✔️ 
- App working as expected ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**